components/Snackbar/src/private/MDCSnackbarOverlayView.m

```diff
@@ -246,7 +246,7 @@ - (void)setSnackbarView:(MDCSnackbarMessageView *)snackbarView {
 
     CGFloat bottomMargin = [self staticBottomMargin];
     CGFloat sideMargin = [self sideMargin];
-    BOOL fullWidth = UI_USER_INTERFACE_IDIOM() != UIUserInterfaceIdiomPad;
+    BOOL isIphone = UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPhone;
```

Comment:
Or even Regular/Regular if you want to prohibit it in a vertically-compact environment (like a landscape iPhone+).

https://useyourloaf.com/blog/size-classes/

Reply:
I agree that size classes are the way to go--I'll look into it!

Reply:
We don't want to break appearance by changing the width from full to non-full on mobile devices for landscape. Currently the behavior and per spec is to have it full width, i think we should leave it at that rather than making breaking layout changes.